.github/workflows/deploy.yml

```
@@ -88,25 +91,37 @@ jobs:
+          if [ "$ENV" = "prod" ]; then
+            echo "name=origin/main" >> "$GITHUB_OUTPUT"
+          else
+            echo "name=origin/develop" >> "$GITHUB_OUTPUT"
+          fi
+
       - name: Deploy to VM
         env:
           REMOTE_HOST: ${{ secrets.VM_HOST }}
           REMOTE_USER: ${{ secrets.VM_USER || 'anton' }}
+          DEPLOY_BRANCH: ${{ steps.branch.outputs.name }}
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2029:info:15:20: Note that, unescaped, this expands on the client side

(shellcheck)